src/pyscaffoldext/custom_extension/custom_extension.py

```diff
@@ -57,3 +65,13 @@ def add_entry_point(self, struct, opts):
     def get_class_name_from_opts(opts):
         pkg_name = opts["package"]
         return "".join(map(str.capitalize, pkg_name.split("_")))
+
+
+def set_pyscaffoldext_namespace(struct, opts):
+    PYSCAFFOLDEXT_NAMESPACE = "pyscaffoldext"
+    namespace_parameter = opts.get("namespace", None)
+    namespace_with_pyscaffoldext = ".".join([PYSCAFFOLDEXT_NAMESPACE, namespace_parameter]) \
```

Comment:
What happens if someone provides an additional namespace parameter? I would actually fail here and report using the PyScaffold error logger: "Error: The namespace of an extension is `pyscaffoldext` if the user sets the namespace and if it is not `pyscaffoldext`.

Reply:
the idea is that if the user provides a namespace, we put that inside the pyscaffoldext namespace. In the end we are interested that everything is inside pyscaffoldext and that the entry point is correctly specified. Additional levels in the namespace shouldn't be a problem, right?

Reply:
okay, so nested namespaces... why not. Sorry for my misunderstanding. 
@abravalheri what do you think?